app/models/manageiq/providers/base_manager/manager_refresher.rb

```diff
@@ -40,8 +45,11 @@ def collect_inventory_for_targets(ems, targets)
           _log.info("Filtering inventory for #{target.class} [#{target_name}] id: [#{target.id}]...")
 
           if inventory_object_refresh?
-            inventory = builder_class_for(ems.class).build_inventory(ems, target)
-          end
+            begin
+              inventory = builder_class_for(ems.class).build_inventory(ems, target)
+            rescue NameError
+              inventory = inventory_class_for(ems.class).build(ems, target)
```

Comment:
Hey what if we add an alias for build_inventory -> build so we can call this one way while we refactor the providers?